src/js/components/about/AboutSidebar.jsx

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
This seems almost identical to AgencySidebar. Any reason why we couldn't just use that component and make the modifications necessary to accept CSS class name as a prop (or wrap around child content)?

Reply:
Done